__tests__/server.test.js

```diff
@@ -3,70 +3,69 @@
 const supertest = require('supertest');
 const { server } = require('../src/server');
 const mockRequest = supertest(server);
```

Comment:
Look at your server.js file, specifically the server instance you created on line 10 and exported on line 26. What is it named?